test/test_nn_ops.cpp

```diff
@@ -52,31 +52,69 @@ namespace testing {
 // https://github.com/google/googletest/blob/master/googletest/docs/primer.md
 // Use only Tensors and ops::Const() to provide input to the test op
 
-TEST(NNOps, SparseSoftmaxCrossEntropyWithLogits) {
-  Scope root = Scope::NewRootScope();
-  int batch = 10;
-  int num_of_classes = 2;
+// The backword operation for "BiasAdd" on the bias tensor.
```

Comment:
small typo 'backward'. Nice comments! :)

Reply:
Thanks for catching that.